### DISCUSSION TITLE
When is the next release due ?

### DISCUSSION THREAD

**Discussion by barroudjo** [gitextensions/gitextensions#9895] [General]
From what I gathered it should be compatible with a repo in WSL(2), and that would be very helpful for my team !

**Comment by gerhardol** [comment]
No date is set. There is a burndown list (Not entirely updated) https://github.com/gitextensions/gitextensions/discussions/9713

The blocker are some issues in .NET5 that may require a move to .NET6

Earliest estimation is an alpha in a week or so, probably longer though.
(master is mostly stable)

**Reply by RussKie** [reply]
Pretty much these are the blockers https://github.com/gitextensions/gitextensions/issues/9402 and https://github.com/gitextensions/gitextensions/pull/9794. I'm still waiting for the underlying bug(s) to be fixed in ,net6.

**Reply by barroudjo** [reply]
OK, thanks for the explanation. I hope the bug in winforms can be fixed soon.

**Comment by borisdamevin** [comment]
Hi!

Any _(good)_ news about the next release? :)

Thank you.

**Reply by RussKie** [reply]
It's tracked here https://github.com/gitextensions/gitextensions/discussions/9713.
I'd say we're on a verge of a release, but the RL takes precedence for everyone involved.
